### PR TITLE
try to do button html test

### PR DESCRIPTION
**Pull request by ahospodarysko**

### CODE REVIEW

File: src/test/java/com/khospodarysko/ButtonTest.java [hunk 1]
```diff
@@ -1,13 +1,32 @@
...
  * Click on button 5 times and assert that button's text is "Clicked 5"
  */
 public class ButtonTest extends BaseTest {
+
     @Test
     public void testExpectedCondition() {
         driver.get("file://" + absoluteFilePath("button"));
+        try {
+            WebDriverWait wait = new WebDriverWait(driver, 20);
+            wait.until(ExpectedConditions.elementToBeClickable(By.cssSelector("button.hide")));
```
Comment (kostia-hospodarysko) [inline src/test/java/com/khospodarysko/ButtonTest.java:20]: Look inside how `elementToBeClickable` is implemented. Next step is to how to implement your own `ExpectedCondition`, let's say `clickElementUntilText()` - it should click on element until its text becomes "Clicked 5" (think of what it has to accept as parameter).
Has to be `wait.until(MyEC.clickElementUntilText(By.cssSelector("button.hide")), "Clicked 5");`.
Use the same approach as in `elementToBeClickable`.

File: src/test/java/com/khospodarysko/ButtonTest.java [hunk 2]
```diff
@@ -1,13 +1,32 @@
...
+        } catch (Exception e) {
+            // ignore
+        }
+        WebElement button = driver.findElement(By.cssSelector("button.hide"));
+        String butt = "";
+        while(!(butt.equals("Clicked 5"))) {
+            button.click();
```
Comment (kostia-hospodarysko) [inline src/test/java/com/khospodarysko/ButtonTest.java:27]: It is enough to click 5 times and to assert, as while already kind of assert the text, no need `while` here.

**Review by kostia-hospodarysko — CHANGES_REQUESTED** [review]

File: src/test/java/com/khospodarysko/ButtonTest.java (hunk 1 above)
Comment (ahospodarysko) [reply src/test/java/com/khospodarysko/ButtonTest.java:20]: I have to create my own class like - MyExpectedCondition - and write my own method - clickElementUntilText(), this method will be waiting if button with text - Delayed Button - is displayed on the page?

File: src/test/java/com/khospodarysko/ButtonTest.java (hunk 2 above)
Comment (ahospodarysko) [reply src/test/java/com/khospodarysko/ButtonTest.java:27]: And here i just click five times - button.click(), button.click() .... and assert if button has clicked?

File: src/test/java/com/khospodarysko/ButtonTest.java (hunk 1 above)
Comment (ahospodarysko) [reply src/test/java/com/khospodarysko/ButtonTest.java:20]: I cant understand why i need this method - clickElementUntilText() - cause i just wait for button and then click five times and assert if button have text - "Clicked 5". In selenium we already have method - visibilityOfElementWithText

**Pull request merged**